iio/iio_app/iio_app.c

```diff
@@ -47,36 +47,53 @@
 #include "no-os/delay.h"
 #include <stdio.h>
 
+#undef USE_TCP_SOCKET
```

Comment:
Question: what is the point of undefining `USE_TCP_SOCKET` here?

Reply:
I forgot to remove it. It was used just for testing